samples/apps/copilot-chat-app/webapi/CopilotChat/Skills/ChatSkills/ChatSkill.cs

```diff
@@ -79,6 +86,7 @@ public class ChatSkill
         "planner",
         "memoryExtraction"
     };
+    private IOptions<DocumentMemoryOptions> _documentImportOptions;
```

Comment:
Kill this, no?

Reply:
yeah done tao left a similar comment up top